bin/maintengine.py

```diff
@@ -24,17 +24,29 @@
 
 import os.path
 import time
+import logging
+
 import nav.path
-from nav.maintengine import init_logging
+from nav.logs import init_generic_logging
 from nav.maintengine import check_devices_on_maintenance
 
 
+LOG_FORMAT = "[%(asctime)s] [%(levelname)s] [pid=%(process)d %(name)s] %(message)s"
```

Comment:
Just the pid. As I said, these commits seek to preserve the existing situation and make the mess more visible.